src/components/Event/UsersEvents/index.tsx

```diff
@@ -52,12 +53,18 @@ const UsersEvents = observer((props: Props) => {
 
     return (
         <Tabs lazy>
-            <TabItem value='my-events' label='Unveröffentlicht'>
+            <TabItem value='my-events' label={translate({
+                message: 'Not published',
+                id: 'components.event.usersevents.index.tabitem.notpublished',
+                description: 'Text not published'})}>
                 <AddButton />
                 {drafts.length > 0 && (
                     <div className={clsx(styles.card, 'card')}>
                         <div className={clsx('card__header')}>
-                            <h3>Unveröffentlicht</h3>
+                            <h3>{translate({
+                                message: 'Not published',
+                                id: 'components.event.usersevents.index.header.notpublished',
+                                description: 'Th: not published'})}</h3>
```

Comment:
Opening and closing tags should be either on the same line, or aligned.
```suggestion
                            <h3>{
                                translate({
                                    message: 'Not published',
                                    id: 'components.event.usersevents.index.header.notpublished',
                                    description: 'Th: not published'
                                })
                            }</h3>
```